data-transforms/redaction/deploy.sh

```diff
@@ -0,0 +1,3 @@
+export CONFIG=$(cat config.yaml | gzip | base64)
```

Comment:
Would it be simpler to just convert to JSON (or just use a JSON config file and minify it) or are you running into length limits? I'm not sure if this is standard, but I guess it's fine 🤷

Reply:
I wouldn't expect folks to have minify available on the command line, but most should have gzip and base64. I wasn't running into space issues, but it did put the compression in to try to avoid hitting any, especially since the yaml should be quite compressible.

Reply:
Good point, if you expect `jq` to be a common tool, that can be used to minify: `echo '{ "foo": "bar" }' | jq -r tostring`